pkgs/tools/text/mecab/default.nix

```diff
@@ -6,6 +6,10 @@ in
 stdenv.mkDerivation (finalAttrs: ((mecab-base finalAttrs) // {
   pname = "mecab";
 
+  # mecab uses several features that have been removed in C++17.
+  # Force the language mode to C++14, so that it can compile with clang 16.
+  makeFlags = [ "CXXFLAGS=-std=c++14" ];
+
```

Comment:
```suggestion
```

Reply:
I’ve moved `makeFlags` to `base.nix` and pushed an updated PR.